inc/views/base/modcp/modqueue/attachment.twig

```diff
@@ -0,0 +1,12 @@
+{% set row = alt_trow() %}
+<tr>
+    <td class="{{ row }}">
+        <a href="attachment.php?aid={{ attachment.aid }}" target="_blank">{{ attachment.filename }}</a> ({{ attachment.filesize }})<br />
+        <small class="modqueue_meta">{{ lang.post }}: <a href="{{ attachment.link}">{{ attachment.postsubject }}</a></small>
```

Comment:
`{{ attachment.link}` is missing a closing `}`.

Reply:
Fixed